assets/styles/style.css

```
@@ -0,0 +1,27 @@
+*{
```

Please add a space between selector and open block like so:

```suggestion
* {
```